tests/config/pytest_modules.yml

```diff
@@ -43,9 +43,6 @@ artic/guppyplex:
 artic/minion:
   - modules/nf-core/artic/minion/**
   - tests/modules/nf-core/artic/minion/**
-ascat:
```

Comment:
Does anyone know why the CI pytest is triggered despite removal from the config by the CI nf-test is not triggered?

Reply:
Only **pytest-changes** is triggered (not sure if it should run or not); **pytest** doesn't run.

Reply:
Yes my bad! But nf-test should be triggered but is not.

Reply:
Could it be because snapshot is not available yet?

Reply:
But then the linting should fail first and that is also not triggered. Can you run nf-test locally?
`nf-core modules test ascat --profile docker`?

Reply:
The snapshot file is there in the test folder though?

Reply:
Yes, it is

Reply:
> But then the linting should fail first and that is also not triggered. Can you run nf-test locally? `nf-core modules test ascat --profile docker`?

I run the test locally but it fails because `loci_files` and `allele_files` are missing.

Reply:
@famosab could you help me to identify the right `loci_files` and `allele_files` for testing? 

Can we use the files used in Sarek? 
https://github.com/nf-core/sarek/blob/6aeac929c924ba382baa42a0fe969b4e0e753ca9/conf/igenomes.config#L15

Reply:
These should work when they work with sarek - I would go for the latest reference genome: https://github.com/nf-core/sarek/blob/6aeac929c924ba382baa42a0fe969b4e0e753ca9/conf/igenomes.config#L46
Somewhere in the sarek conf it also says that you can setup the reference for ascat yourself and there are some intermediate steps to take but I guess they already did those for igenomes. I think we just have to make sure that the test_data input is also hg38 if we use human data.

Reply:
https://github.com/nf-core/test-datasets/blob/modules/docs/ADD_NEW_DATA.md